lib/matplotlib/backends/backend_ps.py

```diff
@@ -134,6 +134,77 @@ def _move_path_to_path_or_stream(src, dst):
         shutil.move(src, dst, copy_function=shutil.copyfile)
 
 
+def _font_to_ps_type3(font_path, glyph_ids):
+    font = get_font(font_path, hinting_factor=1)
+
+    preamble = """\
+%!PS-Adobe-3.0 Resource-Font
+%%Creator: Converted from TrueType to Type 3 by Matplotlib.
+20 dict begin
+/_d {{bind def}} bind def
+/_m {{moveto}} _d
+/_l {{lineto}} _d
+/_ce {{closepath eofill}} _d
+/_c {{curveto}} _d
+/_sc {{setcachedevice}} _d
+/_e {{exec}} _d
+/FontName /{font_name} def
+/PaintType 0 def
+/FontMatrix [{inv_units_per_em} 0 0 {inv_units_per_em} 0 0] def
+/FontBBox [{bbox}] def
+/FontType 3 def
+/Encoding [{encoding}] def
+/CharStrings {num_glyphs} dict dup begin
+/.notdef 0 def
+""".format(font_name=font.postscript_name,
+           inv_units_per_em=1 / font.units_per_EM,
+           bbox=" ".join(map(str, font.bbox)),
+           encoding=" ".join("/{}".format(font.get_glyph_name(glyph_id))
```

Comment:
Is this safe, or do you need to escape any Postscript special values?

Reply:
this should be safe per https://github.com/adobe-type-tools/agl-specification#6-assigning-glyph-names-in-new-fonts.  We can always revisit if someone finds a font with a bad glyph name...